gtsam/geometry/Cal3Fisheye.cpp

```diff
@@ -53,9 +41,10 @@ Point2 Cal3Fisheye::uncalibrate(const Point2& p, OptionalJacobian<2, 9> H1,
   Vector5 K, T;
   K << 1, k1_, k2_, k3_, k4_;
   T << 1, t2, t4, t6, t8;
-  const double scaling = Scaling(r);
-  const double s = scaling * K.dot(T);
-  const double xd = s * xi, yd = s * yi;
+  T *= t;
+  const double theta_d = K.dot(T);
+  const double scaling = (r > 1e-8) ? theta_d / r : 1.0;
```

Comment:
ref: https://arxiv.org/pdf/1807.08957.pdf
scaling = d(theta) / r
![image](https://user-images.githubusercontent.com/7278867/201641565-3ef1319f-b411-4444-bf4d-4e204d50db3e.png)

Reply:
scaling = d(theta) / r 
in (24)